iiab/iiab
# master <-> 6.3 differences and upgrade path to 6.4

This is the net difference blocking a clean upgrade path. Pull this into master then release-6.3 -> master/relese-6.4 should be a clean upgrade path in the future. This should be done as soon as possible before other changes in master are introduced, specificity is_debian-9 related work.
  
Method used, clone with no depth to get full history so a common commit can be found.
git clone https://github.com/iiab/iiab.git --branch release-6.3 
git checkout -b 6.4-master
git pull origin master
fix roles/httpd/tasks/main.yml
git commit -a -m "merge release-6.3 into master"
git push "repo alias" 6.4-master

How to test
"add my git repo to .git/config" quick hack edit and add
[remote "jv"]
	url = https://github.com/jvonau/iiab.git

git clone https://github.com/iiab/iiab.git --branch release-6.3 --depth 1
git checkout -b 6.4-master
   "add my git repo to .git/config"
    git pull "repo alias from .git/config" 6.4-master
git pull jv 6.4-master
Should be a clean pull, with no conflicts
Should fix from email the master run issue on raspbian.
